example/routes.js

```diff
@@ -40,10 +40,19 @@ module.exports.setup = function(app) {
 
   /**
    * @swagger
+   * tag:
+   *   - name: Login
+   *     description: Login
+   *   - name: Accounts
+   *     description: Accounts
+   */
+
+  /**
+   * @swagger
    * /login:
    *   post:
    *     description: Login to the application
-   *     tag: [Users]
+   *     tag: [Users, Login]
```

Comment:
@efmr, would you mind writing a test for this so we make sure that we don't get a regression in the future?